troposphere/static/js/components/modals/instance/InstanceLaunchWizardModal.jsx

```diff
@@ -462,7 +462,7 @@ const InstanceLaunchWizardModal = React.createClass({
         });
     },
 
-    filterSizeList(sizes, imageVersion) {
+    filterSizeList(provider, sizes, imageVersion) {
```

Comment:
This looks like a botched conflict resolution by me when I merged `lint-jsx`. 

You can see the failure here:
https://github.com/cyverse/troposphere/commit/8c1654641fbd40003121f9aa0591a6de875a562b#diff-42a1b2252dd30e28506805e7b97e784a